src/SB/Game/zMusic.h

```diff
@@ -38,7 +38,7 @@ struct zMusicTrackInfo
 
 void volume_reset();
 void zMusicRefreshVolume();
-S32 getCurrLevelMusicEnum();
+static S32 getCurrLevelMusicEnum();
```

Comment:
This function declaration should be removed from the header since other cpp files won't be able to call this function